modules/test/playwright/pages/style-book-web/StyleBooksPage.ts

```diff
@@ -84,12 +90,31 @@ export class StyleBooksPage {
 		).toBeVisible();
 	}
 
-	async selectFrontendTokenCategory(buttonText: string, category: string) {
-		await this.page.getByRole('button', {name: buttonText}).click();
+	async selectFrontendTokenCategory(category: string) {
+		await this.page
+			.locator('.style-book-editor__sidebar-content .form-control-select')
+			.click();
 
 		await this.page.getByText(category).click();
 	}
 
+	async updateTokenInput(label: string, value: string, section?: string) {
```

Comment:
In this method you call the frontend token just `token`, but in `selectFrontendTokenCategory` you refer to a token as `FrontendToken`

Reply:
I named it based on `updateTokenInputColor` just below it. I can change `selectFrontendTokenCategory` to `selectTokenCategory`.